### ISSUE TITLE
Regular institutional user meeting

### ISSUE THREAD

**Issue by mrocklin** [dask/community#52]
There are a few larger institutions that use Dask heavily, and are engaged in the community in a variety of ways.  It might be helpful to create a space for representatives from these institutions to meet regularly by video.  This would allow them to discuss needs that affect institutions of a certain size, and also avoid PR issues that would arise from them having to write down things in public (which can be a roadblock to communication in some cases).  

This came up in a meeting today that spun out of our regular monthly meeting.  People attending that meeting include folks like @datametrician (NVIDIA) @kkraus14 (NVIDIA) @martindurant (Anaconda) @mmccarty (Capital One) @gforsyth (Capital One) @mrocklin (Coiled) and probably some others that I'm forgetting.  I also think that folks from non-profit organizations like Pangeo (@rabernat @jhamman) might find this interesting.  

I propose that we set up a regular meeting time with a monthly cadence to start at the same time as the Dask Monthly meeting, but staggered by two weeks.  If it's popular for a couple of months then we can increase its cadence.

**Comment by jcrist** [comment]
Makes sense to me (also commenting to follow/participate in said meeting when it happens :)).

**Comment by kkraus14** [comment]
Thanks for writing this up @mrocklin. Also cc @quasiben to make sure he's in the loop as well.

It would be good to get a meeting added to the dask docs similar to this page: https://docs.dask.org/en/latest/support.html

I'm unclear on who owns the existing zoom monthly meeting or the calendar, but would ideally go onto there similarly.

**Comment by mrocklin** [comment]
@TomAugspurger it sounds like it would be useful to start expanding access
to the Dask community calendar.  Do you have thoughts on the right way to
do this?

**Comment by rabernat** [comment]
I would gladly participate in such a meeting.

**Comment by sjperkins** [comment]
I would also like to participate.
